packages/sources/coingecko/src/endpoint/crypto.ts

```diff
@@ -64,21 +65,20 @@ const handleBatchedRequest = (
   jobRunID: string,
   request: AdapterRequest,
   response: AxiosResponse<ResponseSchema>,
-  validator: Validator,
   endpoint: string,
-  idToSymbol: Record<string, string>,
+  idToSymbol: OverrideToOriginalSymbol,
 ) => {
   const payload: [AdapterRequest, number][] = []
-  for (const base in response.data) {
+  for (const base of response.data) {
```

Comment:
Why did this change? This no longer works.

The response.data looks like:
```
{ bitcoin: { usd: 41200 }, ethereum: { usd: 2914.46 } }
```
A for..of loop cannot iterate over that.

Reply:
Fixed & added test